### DISCUSSION TITLE
二代现在有看视频的方法吗

### DISCUSSION THREAD

**Discussion by Tone-cloud** [PenUniverse/PenMods#269] [你问我答]
如题

**Comment by xhdhlb** [comment]
https://baigei.cc/index.php/archives/434/

**Reply by xhdhlb** [reply]
那篇文章说是要penmods1.3.0，但是我注册不上tg。我用1.2.0试了，和ai研究了半天之后还是不能正常使用。但是我发现视频播放功能可以单独拿出来用，具体的你问问ai，我也不记得了，反正不太好搞，如果你需要我可以去翻翻历史记录。

**Reply by Tone-cloud** [reply]
这个我已经知道了，这是很久以前发的了，你不知道的话问我

**Reply by xhdhlb** [reply]
你加了tg群吗

**Reply by Tone-cloud** [reply]
对的

**Reply by xhdhlb** [reply]
先问一下现在penmods1.3是不能外传吗？
还有我遇到个问题就是我词典笔扫不了任何东西，刚开机的几秒可以扫但是后面就不行，我重置了系统也没用

**Reply by xhdhlb** [reply]
又和AI搞了一上午，把扫描的问题修好了

**Reply by Tone-cloud** [reply]
估计你自己琢磨会很久，发个邮箱，我打包给你吧

**Comment by Tone-cloud** [comment]
有的，现在是另一个大佬在弄pr，https://github.com/Lyrecoul这个是他账户，他应该有现在最新的。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年2月21日(周六) 上午10:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods] 二代现在有看视频的方法吗 (Discussion #269)




 
又和AI搞了一上午，把扫描的问题修好了

**Reply by xhdhlb** [reply]
我装好了2.0版本后可以用ydpExt或者类似功能吗

**Reply by Tone-cloud** [reply]
不行 而且这个版本可以用ai了，后续可能会增加别的模型（之前有，但因为代码丢失，没了）

**Reply by Tone-cloud** [reply]
你要发到哪

**Reply by xhdhlb** [reply]
ok，我没发

**Reply by Tone-cloud** [reply]
我得问问大佬能不能发到国内

**Reply by xhdhlb** [reply]
想问下现在penmods更新具体是什么情况，为什么直接就2.0.0了？

**Reply by Tone-cloud** [reply]
单纯命名为2而已，没什么

**Reply by xhdhlb** [reply]
那1.3，1.4和1.5呢？还有AI的配置怎么保存，我改了PenModsConfig.json之后会自动变回原来的。

**Reply by Tone-cloud** [reply]
1.3有，你纠结这么多干嘛，这个就是最新的，人家大佬想怎么命名就怎么命名，你直接在词典笔上输入就保存了

**Reply by xhdhlb** [reply]
好吧，谢谢
